### PR TITLE
Kafkachannel conversion webhook

### LINKED ISSUE #1353: KafkaChannel v1beta1 webhook
Part of https://github.com/knative/eventing-contrib/issues/1350

### PR DESCRIPTION
**Pull request by aliok**
Fixes #1353

## Proposed Changes

- Webhook changes for KafkaChannel conversion
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Comment by aliok** [comment]
let's see if we'll get any RBAC issues

cc @matzew

File: kafka/channel/cmd/webhook/main.go [hunk 1]
```diff
@@ -17,9 +17,12 @@ package main
 
 import (
 	"context"
```
Comment (mattmoor) [inline kafka/channel/cmd/webhook/main.go:19]: Format Go code:
```suggestion
	"context"

```

**Comment by matzew** [comment]
/hold

let's rebase this after the #1371 is in, ok @aliok ?

**Comment by aliok** [comment]
oh, right. Forgot to make this a WIP PR

**Comment by matzew** [comment]
@aliok can u rebase?

**Pull request merged**